liiatools/datasets/s251/data_generator/sample_data.py

```diff
@@ -0,0 +1,294 @@
+import random
+import string
+from datetime import date, datetime, timedelta
+
+from sfdata_stream_parser.events import StartTable, StartRow, Cell, EndRow, EndTable
+
+from liiatools.datasets.s251.lds_s251_clean.configuration import Config
+
+s251_columns = [
+    "Child ID",
+    "Date of birth",
+    "Gender",
+    "Ethnicity",
+    "Disability",
+    "Category of need",
+    "Date of last assessment",
+    "Does the child have an EHCP",
+    "Is the child UASC",
+    "Number of missing episodes in current period of care",
+    "Legal status",
+    "Placement start date",
+    "Placement end date",
+    "Date of start of current care period",
+    "Reason for placement change",
+    "Number of placements in last 12 months",
+    "Number of placements in current care period",
+    "Placement type",
+    "Provider type",
+    "Procurement platform",
+    "Procurement framework",
+    "Ofsted URN",
+    "Home postcode",
+    "Placement postcode",
+    "LA of placement",
+    "Total committed cost accrued in FY to date",
+    "Contribution from social care provider/s",
+    "Contribution from health provider/s",
+    "Contribution from education provider/s",
+]
+
+
+def generate_sample_s251_file():
+    """
+    Generate a sample s251 file
+
+    :return: stream of generators containing information required to create a csv file
+    """
+    yield StartTable(expected_columns=s251_columns)
+    for _ in range(10000):
```

Comment:
Should this be an option sent to the function instead of hard-coded?

Reply:
(the 10,000 value)

Reply:
Yeah it definitely should. Although the generator stuff in this PR is a bit outdated now and there's a new branch which is more in keeping with how we should be generating data. I'm happy just to remove all the data generator stuff from this PR